apps/modernization-ui/src/options/autocompete/ResultedTestsAutocomplete.tsx

```diff
@@ -1,41 +1,35 @@
 import { Autocomplete, AutocompleteSingleProps } from 'design-system/autocomplete';
-import { FindDistinctResultedTestQuery, useFindDistinctResultedTestLazyQuery } from 'generated/graphql/schema';
+import { Option, ResultedTestOptionsService } from 'generated';
 import { Selectable } from 'options/selectable';
 
-const ResultedTestsAutocomplete = (props: Omit<AutocompleteSingleProps, 'resolver'>) => {
-    const [getLocalResultedTests] = useFindDistinctResultedTestLazyQuery();
+const onSelectableResultedTest = (response: Array<Option>) => {
+    return response.map(
+        (data): Selectable => ({
+            name: `${data.name} [${data.value}]`,
+            value: data.value,
+            label: `${data.name} [${data.value}]`
+        })
+    );
+};
```

Comment:
suggestion:  This is iterating over the response in order to display in the desired format.  The data is all there why not change how the option itself is displayed? 

The [multi-select](https://github.com/CDCgov/NEDSS-Modernization/blob/7836fc0fd5393637c9d50584d902543ddc40a287/apps/modernization-ui/src/design-system/autocomplete/multi/AutocompleteMulti.tsx#L32) component allows you to specify how the `Selectable` is rendered using the `asValue` and `asDisplay` properties.  It also provides defaults if the functions aren't provided.

  Could you do the same with `Autocomplete`?  This will be useful for Coded Results also